src/setCompliment.js

```diff
@@ -0,0 +1,13 @@
+export default function setCompliment(a, b){
```

Comment:
Check spacing on this file since you have changed it in the `setIntersection` file.